1.1/translations/funcsrules_de.ttl

```diff
@@ -0,0 +1,453 @@
+    skos:prefLabel "distance"@de ;
+    skos:example spec11:B.2.2.4 .
+
+funcs:ehContains a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die true zurückgibt, wenn das erste Geometrieargument das zweite Geometrieargument räumlich enthält. 
+      
+DE-9IM: T*TFF*FF*"""@de ;
+    skos:prefLabel "contains"@de .
+
+funcs:ehCoveredBy a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die true zurückgibt, wenn das erste Geometrieargument vom zweiten Geometrieargument räumlich abgedeckt wird.
+
+DE-9IM: TFF*TFT**"""@de ;
+    skos:prefLabel "covered by"@de .
+
+funcs:ehCovers a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die true zurückgibt, wenn das erste Geometrieargument das zweite Geometrieargument räumlich abdeckt.
+
+DE-9IM: T*TFT*FF*"""@de ;
+    skos:prefLabel "covers"@de .
+
+funcs:ehDisjoint a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die true zurückgibt, wenn die Eingabegeometrien disjunkt sind. 
+
+DE-9IM: FF*FF****"""@de ;
+    skos:prefLabel "disjoint"@de .
+
+funcs:ehEquals a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die wahr zurückgibt, wenn die eingegebenen Geometrien gleich sind. 
+
+DE-9IM: TFFFTFFFT"""@de ;
+    skos:prefLabel "equals"@de .
+
+funcs:ehInside a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die wahr zurückgibt, wenn das erste Geometrieargument räumlich innerhalb des zweiten Geometriearguments liegt.
+
+DE-9IM: TFF*FFT**"""@de ;
+    skos:prefLabel "inside"@de .
+
+funcs:ehMeet a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die wahr zurückgibt, wenn die eingegebenen Geometrien übereinstimmen.
+
+DE-9IM: FT******* ^ F**T***** ^ F***T****"""@de ;
+    skos:prefLabel "meet"@de .
+
+funcs:ehOverlap a skos:Concept ;
+    skos:definition """Eine Abfragefunktion, die true zurückgibt, wenn sich die Eingabegeometrien überschneiden.
+
+DE-9IM: T*T***T**"""@de ;
+    skos:prefLabel "overlap"@de .
+
+funcs:envelope a skos:Concept ;
+    skos:definition "Eine Abfragefunktion, die das kleinste begrenzende Rechteck der Eingabegeometrie zurückgibt."@de ;
+    skos:prefLabel "envelope"@de .
+
+funcs:geometryN a skos:Concept ;
+    skos:definition "Eine Abfrage-Funktion, die die n-te Geometrie zurückgibt, wenn das angegebene Geometrie-Literal eine GeometryCollection enthält."@de ;
+    skos:prefLabel "geometryN"@de .
+
+funcs:getSRID a skos:Concept ;
+    skos:definition "Eine Abfragefunktion, die den URI des räumlichen Bezugssystems der Eingabegeometrie zurückgibt."@de ;
+    skos:prefLabel "getSRID"@de .
+
+funcs:isEmpty a skos:Concept ;
+    skos:definition "Eine Abfragefunktion, die true zurückgibt, wenn die Eingabegeometrie leer ist."@de ;
+    skos:prefLabel "isEmpty"@de .
+
+funcs:isSimple a skos:Concept ;
+    skos:definition "Eine Abfragefunktion, die true zurückgibt, wenn die Eingabegeometrie eine einfache Geometrie ist."@de ;
+    skos:prefLabel "isSimple"@de .
```

Comment:
As per the comment for `asWKT`. You may choose to translate "is simple" into German or use "is simple", as per the English updated form I will shortly make.